src/main/java/io/xdb/core/client/BasicClient.java

```diff
@@ -0,0 +1,95 @@
+package io.xdb.core.client;
+
+import feign.Feign;
+import feign.FeignException;
+import feign.Retryer;
+import io.xdb.core.ServerErrorDecoder;
+import io.xdb.core.exception.XDBHttpException;
+import io.xdb.core.process.BasicClientProcessOptions;
+import io.xdb.core.process.ProcessOptions;
+import io.xdb.gen.api.ApiClient;
+import io.xdb.gen.api.DefaultApi;
+import io.xdb.gen.models.ProcessExecutionDescribeRequest;
+import io.xdb.gen.models.ProcessExecutionDescribeResponse;
+import io.xdb.gen.models.ProcessExecutionStartRequest;
+import io.xdb.gen.models.ProcessExecutionStartResponse;
+import io.xdb.gen.models.ProcessStartConfig;
+
+/**
+ * {@link BasicClient} serves as a foundational client without a process {@link io.xdb.core.registry}.
+ * It represents the internal implementation of the {@link Client}.
+ * However, it can also be utilized directly if there is a compelling reason, allowing you to invoke APIs on the xdb server with minimal type validation checks, such as process type, queue names, and so forth.
```

Comment:
> with minimal type validation checks

I think it's no type check actually.

Reply:
True. I just copied this descrption from the golang-sdk.

Reply:
Ah I will fix that